lib/niconico/video.rb

```diff
@@ -118,6 +118,11 @@ def video_cookie_jar_file
       end
     end
 
+    def add_to_mylist(mylist_id, description='')
+      api = Niconico::NicoAPI.new(@parent)
```

Comment:
Holding NicoAPI object in Niconico object is better, it makes api token reusable